schema.json

```diff
@@ -124,6 +128,16 @@
                 "aarch64-apple-darwin14"
             ],
             "title": "Triplet"
+        },
+        "FileExtension": {
+            "type": "string",
+            "enum": [
+                "dmg",
+                "exe",
+                "tar.gz",
+                "zip"
+            ],
+            "title": "FileExtension"
```

Comment:
This info is already part of the file name, what's the motivation behind a new field? I'm not opposed to it but want to clarify the reason.

If we want to tell apart the different kinds of archives, perhaps a mime type field is a better solution?

Reply:
Yeah, the motivation is to be able to distinguish different archives (e.g. `.tar.gz` versus `.zip`) without the consumer needing to manually parse the filename. 

I'm not sure that MIME type is the best format. E.g. the MIME type for `.dmg` is just `application/octet-stream`, which doesn't seem super useful.

Reply:
> E.g. the MIME type for .dmg is just application/octet-stream, which doesn't seem super useful.

I don't like dmg files :D 

In that case, I'm not sure if it's more efficient to include it in the json file vs parsing it on each run but let's add the key.